kaiax/interface.go

```diff
@@ -142,6 +142,17 @@ type TxPoolModule interface {
 	// Additional actions to be taken when a new tx arrives at txpool
 	PreAddLocal(*types.Transaction) error
 	PreAddRemote(*types.Transaction) error
+
+	// Additional checks to check if a given transaction should be handled by module.
+	IsModuleTx(tx *types.Transaction) bool
+
+	// Optional actions to check if sender balance is valid for module transaction.
+	// This is mainly used on checking if module transaction be appended to queue.
+	// If nil is returned, existing check perform, otherwise returned fuction perform instead of it.
```

Comment:
```suggestion
	// This is mainly used on checking if module transaction should be appended to queue.
	// If nil is returned, default check (balance > txFee) is performed. Otherwise, the returned function overrides default check.
```

Reply:
fix at https://github.com/kaiachain/kaia/pull/263/commits/c135c8f4a3596e4dd4e79739be49f31afd1c8e44